### ISSUE TITLE
Staging Cleanup: Admin permissions change?

### ISSUE THREAD

**Issue by simholt** [osulp/Scholars-Archive#2093]
I deposited into each worktype to test workflows. While the blue status “button” listed the correct stage of review (Grad School Review, Library Review, Pending Review, Honors College Review) and I could approve/advance as needed from the show page, none of the items showed up in the review queue for me.

After discussion with @wickr, I added myself as a Manager for each Admin Set. Then I could see all the queues.

I thought I was able to see items in the queue without being a Manager — there were even some Grad School Review items in my queue from March tests — since I was part of the Repo Managers group?

From wickr:
> this is from the Samvera docs: "In the Dashboard, click on “Tasks,” then “Review Submissions.” You will see a list of works requiring your review. Repository Admins will see all works in the repository, other reviewers will only see works in Admin Sets for which they are designated Managers."

So:
1. I am not sure what altered the Repo Managers group permissions and what impact it would have on production.
2. It looks like there are new Admin Sets for GTDs and HCTs? I don’t know how that is related to the above but if appropriate, it would be nice if old/unused ones were removed.
3. Whatever the situation, the goal is to have staging and prod match.

**Comment by straleyb** [comment]
The admin permission changed because of this PR
https://github.com/osulp/Scholars-Archive/pull/2048

I see that an unintended consequence of this change was that it overrides the ability to edit something as an admin. I can update this change with a quick fix that add a check to see if a current user is an admin to remedy this. 

Is that a direction we would like to go?

**Comment by simholt** [comment]
@straleyb Yes, admins need to be able to make edits.

**Comment by straleyb** [comment]
https://github.com/osulp/Scholars-Archive/pull/2095 This PR should remedy that. @wickr If you want to take a look at it we can merge it, deploy to staging and do another run through to see if there is anything else we need to QA.

**Comment by wickr** [comment]
Ok 2095 is merged and up on staging, ready for more QA. I think leave this open until it's confirmed that staging looks good.

**Comment by simholt** [comment]
More comment than question, but ---

I am not sure what the PR changed -- I never noticed that my editing permissions disappeared (per Brandon's first comment). I noticed that, even though I was part of the Repository Administrators group, I could not see items in the review queue unless I was listed as a manager, which I did not remember being the case. 

Looking back, I don't think Repo Admin group access changed; I think the new GTDs and HCTs admin sets threw me. So: good? Continuing with that:

> this is from the Samvera docs: "In the Dashboard, click on “Tasks,” then “Review Submissions.” You will see a list of works requiring your review. Repository Admins will see all works in the repository, other reviewers will only see works in Admin Sets for which they are designated Managers."

The above is not how things work on prod or staging. I don't think that's a problem, just an FYI. I removed myself as a Manager from the Dataset admin set on production and none of the datasets in the review queue were visible to me. I added myself back as a Manager and they reappeared. Which means the "Users granted a new role will only gain the role on works that are deposited after that role has been granted" may not work? Or that is cancelled out for members of the Repository Administrators group? I am unclear about what being part of the Repository Administrators group means.

Additionally, in comparing the participant abilities on production vs staging, I realized the one admin set I was NOT listed as a Manager for was EESC. I added myself and a document that had been camping out in the review queue since January appeared.

(Related: I am also not sure how the Depositors participant level functions. I don’t think it actually limits users on Production. The Technical Report admin set is set (wrongly) for NO depositors to be able to add new works. But people add new works to this Admin Set all the time. Same with Conference Proceeding or Journal. This has been bothering me for some time but I don't think it is crucial item that needs to be fixed immediately AND has higher-level questions about worktypes and who should/not be able to deposit what.)

It's easy to get confused in terminology with this -- all the test changes I made were done on individual admin set pages:
![Screen Shot 2020-05-20 at 12 56 00 PM](https://user-images.githubusercontent.com/33705514/82491349-639b0900-9a99-11ea-8a27-43a205271da2.png)
